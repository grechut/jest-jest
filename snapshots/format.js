export function formatCurrency(currency) {
    if (currency == null) { 
        throw new Error('Wrong value');
    }

    if (currency >= 1e6) {
        return `$${(currency / 1e6).toFixed(0)}m`
    }
    return `$${currency.toFixed(2)}`;
}
