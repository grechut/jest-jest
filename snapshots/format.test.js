import { formatCurrency } from './format';

describe('formatCurrency', () => {
    it('should properly format 1 million dollars', () => {
        // expect(formatCurrency(1e6)).toBe('$1000000.00');
        // expect(formatCurrency(1e6)).toMatchSnapshot();
    });
})
