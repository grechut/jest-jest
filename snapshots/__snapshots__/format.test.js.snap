exports[`formatCurrency should fail to format undefined 1`] = `"Wrong value"`;

exports[`formatCurrency should properly format 1 dollar 1`] = `"$1.00"`;

exports[`formatCurrency should properly format 1 million dollars 1`] = `"$1m"`;
